hail/www/references.md

```diff
@@ -1,10 +1,20 @@
 # Hail-Powered Science
 
-The following is an incomplete list of scientific work enabled by Hail. We welcome you to add additional examples by [editing this page directly](https://github.com/hail-is/hail/edit/master/hail/www/references.md), after which we will review the pull request to confirm the addition is valid. Please adhere to the existing formatting conventions.
+The following is an incomplete list of scientific work enabled by Hail.
 
-In addition to software development, the Hail team engages in theoretical, algorithmic, and empirical research inspired by scientific collaboration. Examples include [Loss landscapes of regularized linear autoencoders](https://github.com/danielkunin/Regularized-Linear-Autoencoders), [Secure multi-party linear regression at plaintext speed](https://github.com/jbloom22/DASH), and [A synthetic-diploid benchmark for accurate variant-calling evaluation](https://www.nature.com/articles/s41592-018-0054-7).
+If you use Hail for published work, please cite the software.
 
-*Last updated on February 3rd, 2020 at 11:00 AM EST*
+```
+Hail Team. Hail 0.2.13-81ab564db2b4. https://github.com/hail-is/hail/releases/tag/0.2.13.
```

Comment:
this is wrong -- people should cite either the version they use, or the software generally. This would be OK:

```
Hail Team. Hail 0.2. https://github.com/hail-is/hail
```

Reply:
I think I just updated it to reflect this?

Reply:
yup. one more comment (see below0